### PR TITLE
Added Bento learning tracks with free tutorials

### PR DESCRIPTION
**Pull request by JonHMChan**
These are learning tracks made up of free, curated tutorials that I've personally gone through. It covers foundational understanding of each topic (HTML, CSS, JS, Git) including best practices and common pitfalls of each.

### CODE REVIEW

**Comment by eshellman** [comment]
Hi Jon!
Thanks! Why these 5? I see you have 15 tracks. Have you just added the ones you feel are more or less complete? I looked at JS and Git which look like great additions.

**Comment by JonHMChan** [comment]
I just wanted to add to the existing categories that are currently there. I didn't want to impose if the current categories are what we wanted to stay limited to. If the maintainers really want to add them all, let's do it!

File: free-courses-en.md [hunk 1]
```diff
@@ -180,8 +186,14 @@
 * [Problem Solving With Java](https://www.udacity.com/course/intro-to-java-programming--cs046)
 
 
+### JQuery
```
Comment (eshellman) [inline free-courses-en.md]: I think there's some benefit to mimicking the subject index structure found in free-programming-books.md 

There, the jQuery resources list got so long that they were split into a separate file, so maybe this heading and the ###JS should be structured like

* [JavaScript](#javascript)
  * [Javascript frameworks](#javascript-frameworks)
  * [Node.js](#nodejs)

**Comment by eshellman** [comment]
Happy to support Bento, love what you're doing. Where the tracks are substantially complete, let's add them. I see that the categories for the courses list could use some clean-up; alignment with free-programming-books.md categories is desirable.

File: free-courses-en.md [hunk 2]
```diff
@@ -74,9 +75,10 @@
 * [Shaping up with Angular.js](https://www.codeschool.com/courses/shaping-up-with-angular-js)
 
 
-### Bash
+### Bash / Shell
```
Comment (eshellman) [inline free-courses-en.md]: Bash / Shell seems better than just Bash

File: free-courses-en.md [hunk 3]
```diff
@@ -14,12 +14,13 @@
 * [Data Science](#datascience)
 * [Databases](#databases)
 * [Deep Learning](#deep-learning)
-* [Git](#git)
+* [Git / GitHub](#git--github)
```
Comment (eshellman) [inline free-courses-en.md]: I prefer just Git

File: free-courses-en.md [hunk 4]
```diff
@@ -180,8 +186,14 @@
 * [Problem Solving With Java](https://www.udacity.com/course/intro-to-java-programming--cs046)
 
 
+### JQuery
+
+* [Bento jQuery Track](https://bento.io/topic/jquery) (Bento)
+
+
 ### JS
```
Comment (eshellman) [inline free-courses-en.md]: "JS" isn't really a thing.

File: free-courses-en.md (hunk 1 above)
Comment (JonHMChan) [inline free-courses-en.md]: That totally makes sense - will move.

File: free-courses-en.md (hunk 2 above)
Comment (JonHMChan) [inline free-courses-en.md]: Keeping this.

**Comment by JonHMChan** [comment]
No problem - all of those headings should be fixed. Also, I just realized after you had mentioned it that I never even considered putting these resources in the main books list. If you don't mind, I've added it there as well!

**Comment by JonHMChan** [comment]
@eshellman Done. I'm happy with how things look here, unless you need me to help with anything else :)

**Comment by eshellman** [comment]
No, I don't think these are books.

**Comment by JonHMChan** [comment]
That's fair - will revert.

**Comment by eshellman** [comment]
One way you can help is to keep an eye on PRs for the free courses list and comment on their quality and appropriateness, trying always to encourage participation.

**Comment by JonHMChan** [comment]
Sure thing. I'm happy with how things look here if you are.

**Pull request merged**